idaes/models_extra/column_models/MEAsolvent_column.py

```diff
@@ -0,0 +1,1483 @@
+        init_log.info('Initializing enhancement factor (liquid phase) - degrees_of_freedom = {}'.format(degrees_of_freedom(blk)))
+        blk.yi_MEA.unfix()
+        blk.yi_MEA_subst.unfix()
+        blk.enhancement_factor.unfix()
+        
+        for c in enhancement_factor_constraints:
+            getattr(blk, c).activate()
+        
+        blk.E2_eqn.deactivate()
+        blk.E2_obj.activate()
+        
+        for (t, x), v in blk.E1_eqn.items():
+            if x!=1:
+                iscale.constraint_scaling_transform(
+                v, 100)
+            else:
+                pass
+            
+        for (t, x), v in blk.E3_eqn.items():
+            if x!=1:
+                iscale.constraint_scaling_transform(
+                v, 1)
+            else:
+                pass
+        optarg = {
+            "tol": 1e-7,
+            "max_iter": 500,
+            "bound_push":1e-10}
+        opt.options = optarg
+        opt.solve(blk, tee=True)
+        
+        blk.E2_eqn.activate()
+        for (t, x), v in blk.E2_eqn.items():
+            if x!=1:
+                iscale.constraint_scaling_transform(
+                v, 100)
+            else:
+                pass
+        
+        blk.E2_obj.deactivate()
+        
+        opt.solve(blk, tee=True)
+        
+        # ---------------------------------------------------------------------    
+        init_log.info('Step 12: Heat transfer coefficient constraint')
+        init_log.info('Initializing heat transfer coefficent - degrees_of_freedom = {}'.format(degrees_of_freedom(blk)))    
+        blk.heat_transfer_coeff_base.unfix()
+        blk.h_v_constraint.activate()
+        
+        for x in blk.vapor_phase.length_domain:
+            if x == blk.vapor_phase.length_domain.first():
+                pass
+            else:
+                calculate_variable_from_constraint(blk.heat_transfer_coeff_base[0, x],
+                                                blk.h_v_constraint[0, x])        
+        opt.solve(blk, tee=True)
+
+        # Scale variable
+        for (t, x), v in blk.heat_transfer_coeff_base.items():
+            iscale.set_scaling_factor(v, 1/value(blk.heat_transfer_coeff_base[t, x]))
+            
+        # Scale constraint
+        for (t, x), v in blk.h_v_constraint.items():
+            iscale.constraint_scaling_transform(
+            v, iscale.get_scaling_factor(blk.heat_transfer_coeff_base[t, x]))
+            
+        # ---------------------------------------------------------------------    
+        init_log.info('Step 13: Flooding constraint')
+        init_log.info('Initializing flooding calculations - degrees_of_freedom = {}'.format(degrees_of_freedom(blk)))   
+        
+        blk.floodfraction.unfix()
+        blk.flood_fraction_constr.activate()
+        
+        opt.solve(blk, tee=True)
+        init_log.info('Step 14: Initialization completed - degrees_of_freedom = {}'.format(degrees_of_freedom(blk)))
```

Comment:
1. You are missing the step to unfix the inlet conditions using the flags from the property/control volume initializations.
2. You need to do a final check of the solver status and raise an Initialization error if the conditions are not optimal.